python/ray/data/_internal/execution/interfaces/execution_options.py

```diff
@@ -89,6 +89,11 @@ class ExecutionOptions:
     Attributes:
         resource_limits: Set a soft limit on the resource usage during execution.
             This is not supported in bulk execution mode. Autodetected by default.
+        exclude_resources: Amount of resources to exclude from Ray Data.
+            Set this if you have other workloads running on the same cluster.
+            For Ray Data + Ray Train, this should be automatically set.
+            Note for each resource type, resource_limits and exclude_resources can
```

Comment:
Just comment to update the doc here that resources from DataConfig add to exclude_resources, instead of overwriting.

Reply:
Might be good to add a test for this case also.